earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -43,29 +44,28 @@ sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.
 
 ## Install Git
 
-Install the system default version:
+It's recommended to install a recent version of Git from the [IUS repositories](https://ius.io),
+but the older RedHat or Centos packages can also be used.
 
 ```bash
+
+# To the system default version:
 sudo yum install -y git
-```
 
-Alternatively, install the latest version from the IUS repository (recommended).
-See the [Getting Started](https://ius.io/GettingStarted/) page to find the RPM
-URL for your system. Use this URL below:
+# To Install Git 2.16 on RHEL 6 and Centos 6
```

Comment:
Since we are editing this section,  please move the recommended method above the instructions for system default